tests/mux/test_mux.sh

```diff
@@ -118,6 +118,17 @@ rm $SOCKM $SOCKM.pid
 # check for memory leaks
 if ! [ `uname` = "FreeBSD" ]; then
 	echo "Can't check for memory leaks on `uname`"
+
+	# Kill the upstream server, and poke MUX with a new request to make it die
+	kill `cat $SOCKK.pid`
+	rm $SOCKK $SOCKK.pid
+	kill `cat $SOCKL.pid`
+	rm $SOCKL $SOCKL.pid
+	$TESTMUX $SOCKM 0. 2>/dev/null
```

Comment:
Ah, that's because the "poke mux to make it die" was in the checking-for-memory-leaks path.  I wanted mux to recognize that the kvlds had gone away and shut down cleanly, so that all of its memory would be freed.

If we're not checking for memory leaks, `kill` (which will terminate the process without any cleanup, since we don't catch SIGTERM) is the right way to terminate it.